src/dmd/mars.d

```diff
@@ -2075,9 +2075,6 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
                         case 14_246:
                             params.dtorFields = FeatureState.enabled;
                             break;
-                        case 14_488:
-                            params.vcomplex = true;
-                            break;
```

Comment:
Leave the switch as do-nothing so the transition is smoother for users.
We'll remove it when we make it an error to use complex and imaginary types.

Reply:
I would argue that at least a warning should be outputted here. (@ibuclaw ignore my previous comment)